cluster/gce/configure-vm.sh

```diff
@@ -24,6 +24,26 @@ is_push=$@
 readonly KNOWN_TOKENS_FILE="/srv/salt-overlay/salt/kube-apiserver/known_tokens.csv"
 readonly BASIC_AUTH_FILE="/srv/salt-overlay/salt/kube-apiserver/basic_auth.csv"
 
+function ensure-basic-networking() {
+  # Deal with GCE networking bring-up race. (We rely on DNS for a lot,
+  # and it's just not worth doing a whole lot of startup work if this
+  # isn't ready yet.)
+  until getent hosts metadata.google.internal &>/dev/null; do
```

Comment:
For my own understanding, how long do we expect this to take/does it usually take?